draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -730,7 +734,7 @@ SKEYSEED(n) = prf(SK_d(n-1), SK(n) | Ni | Nr)
                 key exchanges, then the additional key exchanges are performed in a series 
                 of new IKE_FOLLOWUP_KE exchanges that follows the CREATE_CHILD_SA exchange.
                 The IKE_FOLLOWUP_KE exchange is introduced as a dedicated exchange for transferring data of additional key exchanges 
-                following the key exchange performed in the CREATE_CHILD_SA. Its Exchange Type is &lt;TBA by IANA&gt;.
+                following the key exchange performed in the CREATE_CHILD_SA. Its Exchange Type is &lt;44 by IANA&gt;.
```

Comment:
Again, I'd rather remove "by IANA". It is clear that all values are assigned by IANA :-)
The same comment applies to all such cases.